cmd/conformance/posix/main.go

```diff
@@ -84,7 +91,10 @@ func main() {
 	})
 	// Proxy all GET requests to the filesystem as a lightweight file server.
 	// This makes it easier to test this implementation from another machine.
-	http.Handle("GET /", http.FileServer(http.Dir(*storageDir)))
+	fs := http.FileServer(http.Dir(*storageDir))
+	http.Handle("GET /checkpoint", addCacheHeaders("no-cache", fs))
+	http.Handle("GET /tile/", addCacheHeaders("public, max-age=31536000, immutable", fs))
```

Comment:
I'm far from an expert on this, but from reading the [mdn docs on this](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) the other day, it might be fine/better to leave `public` out - it's the default anyway for non-authorized resources, it seems like it's only needed if you want to make resources retrieved when using an `Authorization` header be cached in a public cache.

Reply:
Oh, I copied that from the mysql implementation (c30d02cb4c4d82b0dc2f6c2423815714b12bb559), you might want to change that there too.

Reply:
Aha, will do - thanks!

Reply:
https://github.com/transparency-dev/trillian-tessera/pull/417

Reply:
Done.